deps/AMICI/include/amici/rdata.h

```diff
@@ -387,6 +387,18 @@ class ReturnData {
     /** reporting mode */
     RDataReporting rdata_reporting{RDataReporting::full};
 
+    /**
+     * @brief Set likelihood and chi2 to NaN
+     * (typically after integration failure)
+     */
+    void invalidateLLH();
```

Comment:
Please don't make any changes to the AMICI sub-repo here. We only want to patch AMICI only if absolutely necessary. The rest should be handled in parPE code or adapted in AMICI.

Reply:
already changed it after our conversation, didn't push yet